packages/commonwealth/server/api/external-router.ts

```diff
@@ -109,6 +117,11 @@ const api = {
   createContestMetadata,
   updateContestMetadata,
   cancelContestMetadata,
+  deleteContestMetadata,
+  getContestLog,
+  getFarcasterCasts,
+  farcasterWebhook,
+  getJudgeStatus,
```

Comment:
the queries need to be configured with the security flag... see other queries in this file

Reply:
Please see #12248